frontend/sw360-portlet/src/main/java/com/siemens/sw360/portal/portlets/components/ComponentPortlet.java

```diff
@@ -592,23 +590,46 @@ private void addReleaseBreadcrumb(RenderRequest request, RenderResponse response
 
     private void prepareStandardView(RenderRequest request) throws IOException {
         String searchtext = request.getParameter(KEY_SEARCH_TEXT);
-
         String searchfilter = request.getParameter(KEY_SEARCH_FILTER_TEXT);
 
+        List<Component> componentList = getFilteredComponentList(request);
+
+        Set<String> vendorNames;
+
+        try {
+            vendorNames = thriftClients.makeVendorClient().getAllVendorNames();
+        } catch (TException e) {
+            log.error("Problem retrieving all the Vendor names");
+            vendorNames = Collections.emptySet();
+        }
+
+        List<String> componentTypeNames = Arrays.asList(ComponentType.values())
+                .stream()
+                .map(ct -> ThriftEnumUtils.enumToString(ct))
+                .collect(Collectors.toList());
+
+        request.setAttribute(VENDOR_LIST, new ThriftJsonSerializer().toJson(vendorNames));
+        request.setAttribute(COMPONENT_LIST, componentList);
+        request.setAttribute(KEY_SEARCH_TEXT, searchtext);
```

Comment:
Since `searchtext` is set to
```
searchtext = request.getParameter(KEY_SEARCH_TEXT);
```
it is also set in `request` to the desired value. Further is `searchtext` not used in this function and can be removed.

Same applies for `searchfilter`.